src/components/field/field.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="field">
+    <div class="field-name">{{name}}</div>
+    <div class="field-value">
+      <slot/>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'field',
+  props: ['name']
+}
+</script>
+<style scoped lang='scss'>
+.field {
+  display:flex;
+  justify-content:space-between;
+  padding:8px 0;
+}
+.field-name {
```

Comment:
Instead of these verbose classes seem more reasonable to use nested styles:
```
.field {
  ...

  .name {
    ...
  }

  .value {
    ...
  }
}
```

Reply:
i like when class names give you context. `name` alone is very generic and can mean anything. i agree that here is probably to particular but its a pattern i try to follow in general